bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/PerChannelBookieClient.java

```diff
@@ -1097,7 +1097,7 @@ private void closeInternal(boolean permanent, boolean wait) {
             }
             toClose = channel;
             channel = null;
-            makeWritable();
+            makeUnWritable();
```

Comment:
The writable should be a reflection value of the Channel's isWritable. So we make it default to true. So it should always be true except when the channel is non-writable.